app/src/main/res/values/strings.xml

```diff
@@ -617,6 +617,7 @@
     <string name="no_scheduled_posts">You don\'t have any scheduled posts.</string>
     <string name="no_announcements">There are no announcements.</string>
     <string name="warning_scheduling_interval">Mastodon has a minimum scheduling interval of 5 minutes.</string>
+    <string name="pref_title_show_self_username_disambiguation">Show username when multiple accounts are logged in</string>
```

Comment:
This string is long enough it may be confusing to read. However all simpler strings I can think of are vague. Solutions I can think of:
* "Show username when multiple accounts are in use"?
* We could make this a selector, "Explicitly show username:" "Always" "When multiple accounts are logged in" "Never" this might scan better, but also is complexity I'm not sure anyone needs
Is "username" even the right term here?

Reply:
There is also the possibility to use a "summary" to clarify a setting, e.g. like this:

<img src="https://user-images.githubusercontent.com/10157047/196025135-ff38aa2d-aea9-4436-8143-f08616cdb033.png" width="300" /> <img src="https://user-images.githubusercontent.com/10157047/196025138-0a63024c-fd1a-435f-8dfc-ffa459122b9e.png" width="300" />

Reply:
Oh, I think that's good. Do you think it's a problem that the label says "main toolbar" but the setting applies to main toolbar AND compose view?

I guess people will figure it out quick...

Reply:
Maybe we are thinking too complicated. Not showing the username when there is only one login made sense without options. But now it could just be "show username in toolbars", default off. Maybe someone even wants to see the username with only one account.

Reply:
I like that wording...